tests/acceptance/subscribe/steps/when_steps.py

```diff
@@ -4,12 +4,14 @@
 
 
 @when('I subscribe')
-def step_impl(context: PNContext):
+@async_run_until_complete
+async def step_impl(context: PNContext):
     context.pubnub.subscribe().channels('foo').execute()
 
 
 @when('I subscribe with timetoken {timetoken}')
-def step_impl(context: PNContext, timetoken: str): # noqa F811
+@async_run_until_complete
+async def step_impl(context: PNContext, timetoken: str): # noqa F811
```

Comment:
I'm curious, what it means? `# noqa F811`

Reply:
it disables validation errors upon redeclaration of method. in this file there are more methods called `step_impl`, but they are the content of a `@when` decorator. so TL:DR - 'tis just for linter